examples/pytorch/language-modeling/run_mlm.py

```diff
@@ -154,7 +158,7 @@ class DataTrainingArguments:
         metadata={
             "help": (
                 "The maximum total input sequence length after tokenization. Sequences longer "
-                "than this will be truncated."
+                "than this will be truncated, sequences shorter will be padded."
```

Comment:
That's only if `--pad_to_max_length` is activated along with `--line_by_line`, otherwise there is no padding.